presentation/backend/src/test/PresentationRpcImpl.test.ts

```diff
@@ -75,6 +82,8 @@ describe("PresentationRpcImpl", () => {
 
     const imodelTokenMock = moq.Mock.ofType<IModelRpcProps>();
     const imodelMock = moq.Mock.ofType<IModelDb>();
+    // need to define .then because mocks aren't 'thenable'. async functions which return IModelDb hang without this.
+    imodelMock.setup((x: any) => x.then).returns(() => undefined);
```

Comment:
You could use `configureForPromiseResult` helper function for this. https://github.com/iTwin/itwinjs-core/blob/8ef85dabc4b5665c69aa1b3abeacc594c66cb314/presentation/common/src/test/_helpers/Mocks.ts#L16

Reply:
Thanks, changed to use that.